MAUI/Circular-Charts/DataLabels.md

```diff
@@ -222,4 +300,76 @@ this.Content = chart;
 
 {% endtabs %}
 
-![Label template in MAUI Chart](DataLabel_images/maui_chart_datalabel_template.png)
\ No newline at end of file
+![Label template in MAUI Chart](DataLabel_images/maui_chart_datalabel_template.png)
+
+## Smart Labels
+
+This feature is used to arrange the data labels smartly and avoid the intersection when there is overlapping of labels. The [SmartLabelAlignment]() enum contains the following values.
```

Comment:
@EzhilarasanElangovan31  In MAUI, we don't mention as feature in the content. We will be using the property and explain its usage. Check MAUI UG and modify.

Reply:
Modified the content